### DISCUSSION TITLE
Mousewheel Volume Control

### DISCUSSION THREAD

**Discussion by GipfelSchema64** [tidal-music/discussions#112] [Ideas & Feature requests]
Please add feture, to control the "in-app"-loudness with the mousewheel by simply hover over the volume icon. :)

**Comment by stefanmoro** [comment]
This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com/) (TIDAL API/SDK, open source projects etc).

As this question relates to an existing product offering, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).

Thanks
